app/src/main/java/org/wikipedia/views/ObservableWebView.kt

```diff
@@ -135,8 +135,10 @@ class ObservableWebView : WebView {
             MotionEvent.ACTION_UP -> {
                 if (abs(event.x - touchStartX) <= touchSlop &&
                         abs(event.y - touchStartY) <= touchSlop) {
-                    if (onClickListeners.any { it.onClick(event.x, event.y) }) {
-                        return true
+                    if (hitTestResult.type == HitTestResult.UNKNOWN_TYPE) {
```

Comment:
Could you please add a note here to explain why we should check here and why `HitTestResult.UNKNOWN_TYPE`?

Reply:
done!